internal/confmapprovider/discovery/bundle/bundle.d/receivers/istio.discovery.yaml.tmpl

```diff
@@ -0,0 +1,34 @@
+{{ receiver "prometheus/istio" }}:
+  enabled: true
+  rule:
+    k8s_observer: type == "pod" and ("istio.io/rev" in annotations or name matches "istio.*")
+  config:
+    default:
+      config:
+        scrape_configs:
+          - job_name: 'istio'
```

Comment:
This shows up as `service.name` in the final metrics. Any feedback on a better name is welcome.